packages/styles/breadcrumb.css

```diff
@@ -7,8 +7,13 @@
   margin: 0;
 }
 
-.Breadcrumb__Separator {
-  color: var(--gray-40);
+.cauldron--theme-light .Breadcrumb__Separator {
```

Comment:
Mostly looks good. We need to split out the changes so that `padding` is only listed once.

```css
.BreadCrumb__Separator {
  padding: ...
}

.cauldron--theme-dark { ... }
```

The default mode is light, so the color doesn't have to be set explicitly there.

Reply:
Yep - good catch!